### PR TITLE
ENH Add cuml solver

### LINKED ISSUE #60: add cuml solver
https://docs.rapids.ai/api/cuml/stable/api.html#lasso-regression

### PR DESCRIPTION
**Pull request by tanglef**
Fixes #60  by adding `cuml` solver (only available with GPU hardware)

### CODE REVIEW

File: solvers/cuml.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+from benchopt import BaseSolver, safe_import_context
+from benchopt.utils.sys_info import _get_cuda_version
+
+
+with safe_import_context() as import_ctx:
+    cuda_version = _get_cuda_version()
+    if cuda_version is not None:
+        cuda_version = cuda_version.split("cuda_", 1)[1][:4]
+    if cuda_version is None:
```
Comment (agramfort) [inline solvers/cuml.py:9]:
```suggestion
    else:
```

**Comment by tomMoral** [comment]
Did you run it? how good are the results?

**Comment by tanglef** [comment]
> Did you run it? how good are the results?

There's a discussion about the results in the associated issue

**Comment by mathurinm** [comment]
@tanglef ready to merge?

**Comment by tanglef** [comment]
> @tanglef ready to merge?
Yes, sorry I forgot to do the cuda version change here. Thanks

**Comment by mathurinm** [comment]
No problem, thanks for the cool work!

**Pull request merged**